components/sdgCard.vue

```diff
@@ -1,28 +1,39 @@
 <template>
-  <section class="col-md-3 col-sm-6 m-auto w-400">
-    <div class="our-team w-70 float-left relative p-10">
+  <section class="">
+    <div
+      class="
+        our-team
+        w-60
+        float-left
+        m-auto
+        text-center
+        relative
+        overflow-hidden
+      "
+    >
+      <img :src="imageUrl" alt="profile" class="w-100% h-auto" />
       <div
         class="
-          pic
-          rounded-l-3xl
-          overflow-hidden
-          border-t-5 border-b-5 border-gray-600
-          hover:border-hex-8f2d56 hover:transition-colors
+          team-content
+          m-auto
+          w-100%
+          h-100%
+          bg-hex-640087
```

Comment:
Would make more sense to keep it separately as a new class otherwise isn't prominent enough